src/components/webView/iframe.native.js

```diff
@@ -7,12 +7,13 @@ import Section from '../common/layout/Section'
 import Icon from '../common/view/Icon'
 
 import { getMaxDeviceHeight } from '../../lib/utils/Orientation'
+import useOnPress from '../../lib/hooks/useOnPress'
 import useLoadingIndicator from '../../lib/hooks/useLoadingIndicator'
 
 import embed from './iframe.embed.html'
 
 const wHeight = getMaxDeviceHeight()
-const DOMLoadedDispatcher = embed.replace(/<script.+?>/g, '')
+const DOMLoadedDispatcher = String(embed).replace(/<script.+?>/g, '')
```

Comment:
did it was solved ? is it imports 'iframe.embed.html' ? or you just declared

```javascript
const DOMLoadedDispatcher = `
(function () {
    var DOMReady = 'DOMContentLoaded';
    var messenger = (window.ReactNativeWebView || parent || {}).postMessage;

    if ('function' === (typeof messenger)) {
      return;
    }

    window.addEventListener(DOMReady, function() {
      var messagePayload = {
        event: DOMReady,
        target: 'iframe',
        src: location.href
      };

      parent.postMessage(messagePayload, '*')
    });
  })()
`
```

as i've advised in the Slack ?

how did you solved this issue ?

Reply:
I'll remove this changes and I'll focus on this problem on other PR.